Marlin/src/lcd/dogm/dogm_Statusscreen.h

```diff
@@ -1367,14 +1367,18 @@
 #endif
 #if STATUS_LOGO_WIDTH
   #ifndef STATUS_LOGO_X
-    #define STATUS_LOGO_X 0
-  #endif
-  #ifndef STATUS_LOGO_Y
-    #define STATUS_LOGO_Y _MIN(0U, (20 - (STATUS_LOGO_HEIGHT)) / 2)
+    #ifndef STATUS_HEATERS_X
+      #define STATUS_LOGO_X 0
+    #else
+      #define STATUS_LOGO_X (STATUS_HEATERS_X - STATUS_LOGO_WIDTH - 1) / 2
+    #endif
   #endif
   #ifndef STATUS_LOGO_HEIGHT
     #define STATUS_LOGO_HEIGHT (sizeof(status_logo_bmp) / (STATUS_LOGO_BYTEWIDTH))
   #endif
+  #ifndef STATUS_LOGO_Y
+    #define STATUS_LOGO_Y _MAX(0U, (14 - ( _MIN(28U, STATUS_LOGO_HEIGHT) / 2)))
```

Comment:
Are the 14/28 constants something that can be pulled from existing constants? It feels like they are probably being defined in multiple places with this change.

Reply:
This thing is tailored to the standard DOGM layout, so 28 tall is about right for the top area. We can/should make different sets of these for the higher resolution and color screens to adapt things to to their dimensions.